### PR TITLE
[ContainerApp]: .NET SDK generated for API version "2022-03-01"

### PR DESCRIPTION
**Pull request by Kotasudhakarreddy**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Comment by panchagnula** [comment]
@ArthurMa1978 given this is a new RP - should this be released using Track2? If so what are the instructions for the same?

**Comment by Kotasudhakarreddy** [comment]
@ruslany @anthonychu - FYI.

**Comment by Kotasudhakarreddy** [comment]
@ArthurMa1978 could you please review and merge to the main branch if everything is fine. We need to release this build by 23rd  May. Thanks.

**Comment by panchagnula** [comment]
> @ArthurMa1978 could you please review and merge to the main branch if everything is fine. We need to release this build by 23rd May. Thanks.

@ArthurMa1978 we'd like to have this released & available for //BUILD, since this service is GAing during this time.

**Review by panchagnula — APPROVED** [review]
LGTM

File: eng/mgmt/mgmtmetadata/app_resource-manager.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿Installing AutoRest version: v2
+AutoRest installed successfully.
+Commencing code generation
+Generating CSharp code
+Executing AutoRest command
+cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/main/specification/app/resource-manager/readme.md --csharp --version=2.0.4421 --reflect-api-versions --csharp-sdks-folder=D:\DotNet-SDK\May-2022\azure-sdk-for-net\sdk
```
Comment (ArthurMa1978) [inline eng/mgmt/mgmtmetadata/app_resource-manager.txt:6]: All Track 1 SDKs will be deprecated soon As the Track 2 SDK has released stable (GA) version. So from now any Track 1 SDK release PR will now be approved and release to Nuget, please contact us if you still need a Track 1 release and ask for an exemption.
And here for ContainerApps there already have a beta version https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/containerapps/Azure.ResourceManager.Applications.Containers

**Comment by ArthurMa1978** [comment]
> > @ArthurMa1978
> >             
> >             Arthur Ma
> >             FTE could you please review and merge to the main branch if everything is fine. We need to release this build by 23rd May. Thanks.
> 
> @ArthurMa1978 Arthur Ma FTE we'd like to have this released & available for //BUILD, since this service is GAing during this time.

Who will use this .net SDK, internal or external customer?

File: eng/mgmt/mgmtmetadata/app_resource-manager.txt (hunk 1 above)
Comment (panchagnula) [reply eng/mgmt/mgmtmetadata/app_resource-manager.txt:6]: @ArthurMa1978 we want to release this for //BUILD. Is the beta version of track2 you linked using the latest swagger changes & stable version. if not can we get this released for //BUILD & we will work on updating the Track2 version  as well?

Comment (ruslany) [reply eng/mgmt/mgmtmetadata/app_resource-manager.txt:6]: @panchagnula , @ArthurMa1978 - the pre-release does not look up-to-date. The parameter names do not match the azure-rest-api-spec. For example:

Method to deactivate revision uses "name" as the revision name parameter: https://github.com/Azure/azure-sdk-for-net/blob/3046996080441d92a3137042add76b3d8dcc5759/sdk/containerapps/Azure.ResourceManager.Applications.Containers/src/Generated/RestOperations/ContainerAppsRevisionsRestOperations.cs#L345

However the spec uses "revisionName": https://github.com/Azure/azure-rest-api-specs/blob/b9d9355460992ba575206a96db8556985f579d9d/specification/app/resource-manager/Microsoft.App/stable/2022-03-01/ContainerAppsRevisions.json#L178

**Comment by Kotasudhakarreddy** [comment]
closed as new PR has been raised using Track2

**Pull request closed without merging**